portal-api/src/auth/login-handle.ts

```diff
@@ -8,6 +9,7 @@ export const providerLoginHandler = async (userInfo: UserInfo, done) => {
       done(null, user);
     })
     .catch((err) => {
+      logApp.error('Provider login error: ' + err);
```

Comment:
```suggestion
      logApp.error(err);
```
Pass the Error object to have a better handling.
If we want to add "Provider login error" to identify the error type, you can add a `AppLogsCategory` enum value like `LOGIN_PROVIDER`.